extra/exports

```diff
@@ -362,6 +362,7 @@ lua_upvalueid
 lua_upvaluejoin
 lua_xmove
 lua_yield
+interval_unpack
```

Comment:
`i` goes before `l` (the file should be sorted alphabetically). But what is more important - we shouldn't add new symbols without `tnt_` prefix. Otherwise they might clash with user's symbols. The old code will have to live with the old names, but in all the new code we need `tnt_`.

Reply:
Thanks, fixed.